bamazonCustomer.js

```diff
@@ -0,0 +1,146 @@
+var inquirer = require("inquirer"); // User input from the terminal
+var mysql = require("mysql"); // MySQL database 
+require("console.table"); // Print Tables
+
+
+// Connection parameters for the MySQL database
+var connection = mysql.createConnection({
+	host: "127.0.0.1",
+	port: 3306,
+	user: "root",
+	password: "",
+	database: "bamazon_db"
+});
+
+// Pretty Print database entries in a table format using console.table 
+function prettyPrintAllProducts(response) {
+	var columns = ['ID', 'Product', 'Department', 'Price ($)', 'Stock'];
+	var entries = [];
+
+	for (var i=0; i<response.length; i++) {
+		var item = [];
+		item.push(response[i].id);
+		item.push(response[i].product);
+		item.push(response[i].department);
+		item.push(response[i].price);
+		item.push(response[i].stock_quantity);
+		entries.push(item);
+	}
```

Comment:
this could be rewritten using map:
```javascript
var entries = response.map(function(item){
  return [item.id, item.product, item.department, item.price, item.stock_quantity] 
})